skim/python/skimReco.py

```diff
@@ -17,17 +17,89 @@
 #process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 #from Configuration.AlCa.GlobalTag import GlobalTag
 #process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_mc', '')
-process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
-process.GlobalTag.globaltag = 'GR_P_V56::All'
+#process.GlobalTag.globaltag = 'GR_P_V56::All'
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
+process.GlobalTag.globaltag = '74X_dataRun2_Prompt_v0'
```

Comment:
I checked with Robert and Maria, and we agree that this GlobalTag is up-to-date.

Cheers,
C.


Am 30.09.2015 um 14:05 schrieb mmarionncern <notifications@github.com>:

> In skim/python/skimReco.py:
> 
> > @@ -17,17 +17,89 @@
> >  #process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
> >  #from Configuration.AlCa.GlobalTag import GlobalTag
> >  #process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_mc', '')
> > -process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
> > -process.GlobalTag.globaltag = 'GR_P_V56::All'
> > +#process.GlobalTag.globaltag = 'GR_P_V56::All'
> > +process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
> > +process.GlobalTag.globaltag = '74X_dataRun2_Prompt_v0'
> same question here
> 
> —
> Reply to this email directly or view it on GitHub.
>